tests/tests_pytorch/accelerators/test_accelerator_connector.py

```diff
@@ -330,12 +326,6 @@ def test_accelerator_auto_with_devices_gpu():
     assert trainer.num_devices == 1
 
 
-def test_validate_accelerator_and_devices():
```

Comment:
Are you removing this because it's duplicated?

Reply:
No but because now it would raise an error. This test seems like a forgotten artifact that silently passed because ddp_cpu was converted to just cpu. Now it is considered invalid and will raise.